modules/guides/pages/bulk-operations.adoc

```diff
@@ -138,7 +138,7 @@ To read multiple documents in Couchbase perform a bulk `get` operation.
 
 [{tabs}]
 ====
-ifdef::command-line[]
+ifdef::flag-devex-command-line[]
```

Comment:
Just curious what we're using these tags for?

Reply:
I'd ask Daniel exactly what the tag refers to, but I'm using them to remove references to the cbq shell as you currently can't use that with elixir